auto-check-deletion/check-deletion.sh

```diff
@@ -1,33 +1,31 @@
 #!/bin/bash
 
-filtered_tests=$(cat filtered_tests.txt | sed 's/,.*\//,/' | rev | cut -d'.' -f 1,2 | rev);
+filtered_tests=$(cat filtered_tests.txt | sed 's/,.*\//,/' | rev | cut -d'.' -f 1,2 | rev)
 
-output_file='check-deletion-output.txt';
+output_file='check-deletion-output.txt'
 
-# Check the file is exists or not
-if [ -f $output_file ]; then
-   rm check-deletion-output.txt;
+if [ -f "$output_file" ]; then
```

Comment:
Why not always `rm` the file, with `-f` if you want to avoid warning about a non-existent file.